### PR TITLE
Build manual

### PR DESCRIPTION
**Pull request by kyrsjo**
Working on the build manual. @JamesMolson , you're welcome to contribute -- especially if you could fill out the CDASH section? I'll be writing something more in Chapter 1.1 soon (I have the text 95% done in another document).

### CODE REVIEW

File: Doc/building_sixtrack/building_sixtrack.tex [hunk 1]
```diff
@@ -204,6 +204,13 @@ \subsubsection{Increased particle numbers}
 
 \todo[inline]{Reference sixtrack meeting slides?}
 
+\subsubsection{DA (differential algebra)}
+This version allows computing DA maps of the machine.
+Please contact Frank Schmidt if you intend to use it.
+
+To compile it, simply use \texttt{cmake\_six DA NAGLIB}~.
+Note that the DA version requires the NAGFOR library (Section~\ref{sec:building:libs:naglib}), a proprietary library of mathematical that is available at CERN via AFS.
```
Comment (JamesMolson) [inline Doc/building_sixtrack/building_sixtrack.tex]: of mathematical?

File: Doc/building_sixtrack/building_sixtrack.tex [hunk 2]
```diff
@@ -270,7 +285,9 @@ \subsection{Running CTEST}
 This will iterate over the tests, running SixTrack for each of them, and check the output for pass/fail.
 
 To run several tests in parallel, use the flag \texttt{-j \textit{numjobs}}.
+
 To run specific tests, use the flag \texttt{-R \textit{name}}, where \textit{name} is a regexp contained in the names of the tests you want to run.
+The tests are classified as \texttt{fast} ($\lesssim 30~\mathrm{secconds}$), \texttt{medium} ($\lesssim 30~\mathrm{minutes}$), or \texttt{slow}; it is possible to run a group by using the \texttt{-L} flag, e.g.\ \texttt{-L fast}~.
```
Comment (JamesMolson) [inline Doc/building_sixtrack/building_sixtrack.tex]: < 60 seconds :)

File: Doc/building_sixtrack/building_sixtrack.tex (hunk 1 above)
Comment (kyrsjo) [inline Doc/building_sixtrack/building_sixtrack.tex]: functions!

thanks.

File: Doc/building_sixtrack/building_sixtrack.tex (hunk 2 above)
Comment (kyrsjo) [inline Doc/building_sixtrack/building_sixtrack.tex]: Sure? In https://github.com/SixTrack/SixTrack/pull/162 you wrote 30 minutes...

Comment (JamesMolson) [inline Doc/building_sixtrack/building_sixtrack.tex]: ($\lesssim 30~\mathrm{secconds}$)


that

Comment (kyrsjo) [inline Doc/building_sixtrack/building_sixtrack.tex]: ah. Apparently I can't read.

*fixing*.

**Pull request merged**